### ISSUE TITLE
Allow admin to determine how the persitent ID field maps to RDF

### ISSUE THREAD

**Issue by mjordan** [mjordan/persistent_identifiers#8]
schema:sameAs is a good default. This module can then implement a hook (`hook_jsonld_alter_normalized_array`?) to emit the RDF.

**Comment by mjordan** [comment]
Resolved with ad7b5dbfefca76de3074c7e80942d788436bda57. Admin can choose to map it to JSON-LD, and what to prepend the PI with, but not what property it's mapped to (it's mapped to schema:sameAs).

**Comment by mjordan** [comment]
Repoening, it appears this option doesn't do anything.

**Comment by mjordan** [comment]
Currently, by default Islandora emits a SameAs that has property that has the node URL as its object:

```
   "http://schema.org/sameAs": [
        {
          "@id": "https://future.islandora.ca/node/8"
        }
      ]
```
README says this option is supposed to emit another SameAs with the first identifier as it's object, e.g.:

```
   "http://schema.org/sameAs": [
        {
          "@id": "https://future.islandora.ca/node/8"
        },
        {
          "@id": "https://ids-n2t2-stg.n2t.net/ark:/99999/fq3js30"
        }
      ]
```
We also need better documentation as to what the "Resolver base URL" does - for PIDs that do not start with and `http` URL like DOIs, it provides a way to prepend a resolver URL.

**Comment by mjordan** [comment]
Resolved with 84e9f5ce311e14c80ef4baba1399919e0c959f34.
